sale_order_invoice_address_propose/views/sale_order_views.xml

```diff
@@ -18,4 +18,15 @@
         </field>
     </record>
 
+    <record id="view_order_form_inherit_sale_view_adj_hls" model="ir.ui.view">
+        <field name="name">sale.order.form.inherit.sale.view.adj.hls</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale_view_adj_hls.view_order_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='partner_invoice_id'][2]" position="attributes">
```

Comment:
Is it necessary to have `[2]` here?

Reply:
Yes, without [2], it won't add the domain to partner_invoice_id.